src/test/resources/eager/handles-complex-raw.jinja

```diff
@@ -0,0 +1,10 @@
+{%- macro raw_exp(var) -%}
+{{ '{% raw %}}} ' ~ var ~ ' {{{% endraw %}' }}
+{%- endmacro -%}
+{%- macro raw_print(var) -%}
+{% print '}} ' ~ var ~ ' {{' %}
+{%- endmacro -%}
```

Comment:
This is a difficult test to solve for so, but I want there to be parity between default and eager execution, so this should be handled. It's handled by re-rendering the result of a macro tag after it's output if raw tags are being preserved.